src/components/Button/Button.vue

```diff
@@ -548,36 +497,13 @@ export default {
 		width: 100%;
 	}
 
-	// We use box-shadow around our buttons instead of an outline, so that the added "border"
-	// coincides with the border of the element. It's not possible to add a border-radius to
-	// the outline
-	&--tabbed, &:focus-visible {
-		box-shadow: 0 0 0 2px var(--color-main-text);
-		background-color: var(--color-primary-light-hover);
-		&.button-vue--vue-primary {
-			background-color: var(--color-primary-hover);
-		}
-		&.button-vue--vue-secondary {
-			box-shadow: 0 0 0 2px var(--color-main-text);
-		}
-		&.button-vue--vue-tertiary-no-background {
-			opacity: 1;
-		}
+	&:focus-visible {
+		outline: 2px solid var(--color-main-text) !important;
```

Comment:
On server we are using a primary color outline for a lot of elements, so wondering if using main-text is intended here?

Reply:
Ref: https://github.com/nextcloud/server/pull/31584

Reply:
Yes this has been the case for the button component since the beginning

Reply:
All right, just seen this is tracked separately with https://github.com/nextcloud/nextcloud-vue/issues/2824